understanding/dragging-movements.html

```diff
@@ -56,213 +56,154 @@
-                     features to be keyboard accessible. However, achieving keyboard equivalence for a
-                     dragging operation does not automatically meet this Success Criterion. It is possible
-                     to create an interface that works with dragging and keyboard controls that does not
-                     work using only clicks or taps. While many designs can be created for a dragging alternative
-                     which address both keyboard accessibility and operability by single pointer operation,
-                     the two requirements should be assessed independently.
+                  <p>達成基準 2.1.1 キーボード及び 2.1.3 キーボード (例外なし) は、ドラッグ機能がキーボードでアクセスできることを必要とする。しかし、ドラッグ操作に対してキーボードの同等性を達成しても、自動的にはこの達成基準を満たさない。クリック又はタップだけでは動作しない、ドラッグ又はキーボードコントロールと連動するインタフェースを作成することは可能である。キーボードのアクセシビリティとシングルポインタ操作による操作性の両方に対処するドラッグの代替として多くの設計を作成できるが、二つの要件は個別に評価されるべきである。
                   </p>
                   
                   
-                  <p>This Success Criterion applies to dragging movements as opposed to pointer gestures,
-                     which are covered in <a href="pointer-gestures">Success Criterion 2.5.1 Pointer Gestures</a>. Pointer gestures include directional path-based as well as multi-point gestures.
-                     In contrast, for dragging movements, only the start and end point of the movement
-                     matters, not the actual path.
+                  <p>この達成基準は、<a href="pointer-gestures">達成基準 2.5.1 ポインタのジェスチャ</a>で説明されているポインタジェスチャではなく、ドラッグ動作に適用される。ポインタジェスチャには、方向性のある軌跡ベースのジェスチャだけでなくマルチポイントのジェスチャも含まれる。対照的に、ドラッグ動作の場合は、実際の軌跡ではなく、動作の開始点と終了点だけが問題になる。
                   </p>
                   
                   
-                  <p>Additional examples are selection rectangles that set the first x/y rectangle coordinate
-                     at the pointer position via a pointer down-event, and the second x/y coordinate, after
-                     a dragging movement, at the next up-event. A similar example is a connecting line
-                     drawn between two different items on the screen, as in an allocation test where users
-                     are required to draw a line between questions and corresponding answers. In these
-                     cases, the dragging movement requires an alternative way to accomplish the same action
-                     that does not rely on the dragging movement. For example, two separate single tap
-                     or click actions may define the rectangle coordinates or the start and end points
-                     of a connecting line.
+                  <p>追加の例としては、ポインタのダウンイベントによってそのポインタの位置に 1 番目の x/y 直交座標を設定し、ドラッグ移動後の次のアップイベントで 2 番目の x/y 座標を設定する選択範囲がある。同様の例は、利用者が質問と対応する回答の間に線を引くように要求される割り当てテストのように、画面上の二つの異なる項目の間に引かれる接続線である。このような場合、ドラッグ動作には、ドラッグ動作に依存しない同じアクションを達成する別の方法が必要である。例えば、二つの別々のシングルタップ又はクリック操作で、直交座標、又は接続線の始点及び終点を定義できる。
```

Comment:
rectangle coordinate を直交座標としていますが、ここでは単に四角形の (頂点の) 座標という意味かと思います。
A similar exampleは同様なわけではないので「似た例」くらいにしたい気がします。
こんな感じでしょうか。

```suggestion
                  <p>追加の例としては、ポインタのダウンイベントによってそのポインタの位置に四角形の 1 番目の x/y 座標を設定し、ドラッグ移動後の次のアップイベントで 2 番目の x/y 座標を設定する選択範囲がある。似た例として、利用者が質問と対応する回答の間に線を引くように要求される割り当てテストのように、画面上の二つの異なる項目の間に引かれる接続線がある。このような場合、ドラッグ動作には、ドラッグ動作に依存しない同じアクションを達成する別の方法が必要である。例えば、二つの別々のシングルタップ又はクリック操作で、四角形の座標、又は接続線の始点及び終点を定義できる。
```